src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMethodImportCompletionHelper.ExtensionMethodSymbolComputer.cs

```diff
@@ -65,79 +72,91 @@ private class ExtensionMethodSymbolComputer
                     document, semanticModel, receiverTypeSymbol, position, namespaceInScope);
             }
 
+            private static string? GetPEReferenceCacheKey(PortableExecutableReference peReference)
+                => peReference.FilePath ?? peReference.Display;
+
             /// <summary>
-            /// Force create all relevant indices
+            /// Force create/update all relevant indices
             /// </summary>
-            public static Task PopulateIndicesAsync(Project? project, IImportCompletionCacheService<CacheEntry, object> cacheService, CancellationToken cancellationToken)
+            public static Task UpdateCacheAsync(Project? project)
             {
-                if (project is null)
+                if (project is not null)
```

Comment:
when is the project actually null?  seems... strange :)

Reply:
removed.